src/overlays/actors/ovl_En_Invadepoh/z_en_invadepoh.c

```diff
@@ -20,492 +243,4858 @@ const ActorInit En_Invadepoh_InitVars = {
+        temp_v1_2 *= 0.85f;
+        temp_v1_2 -= this->actor.shape.rot.x;
+        substruct->unk26.x = CLAMP(temp_v1_2, -0x9C4, 0x9C4);
+        diff = (s16)(this->actor.yawTowardsPlayer - this->actor.shape.rot.y);
+        temp_v1_2 = diff;
+        temp_v1_2 *= 0.7f;
+        substruct->unk26.y = CLAMP(temp_v1_2, -0x1F40, 0x1F40);
+        if ((player->unk840 & 0xFF) == 0) {
+            substruct->unk26.z = Rand_S16Offset(-0x5DC, 0xBB8);
+        }
+    } else {
+        substruct->unk26.x = 0;
+        substruct->unk26.y = 0;
+        substruct->unk26.z = 0;
+    }
+    if (this->actionTimer > 0) {
+        this->actionTimer--;
+    } else {
+        func_80B4A570(this);
+    }
+}
+#else
 #pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Invadepoh_0x80B439B0/func_80B4A350.asm")
-
+#endif
+
+// ISMATCHING: Move rodata once all funcs match
+#ifdef NON_MATCHING
+void func_80B4A570(EnInvadepoh* this) {
+    AlienBehaviorInfo* substruct = &this->behaviorInfo;
+
+    SkelAnime_ChangeAnimTransitionStop(&this->skelAnime, &D_06002A8C, -10.0f);
+    substruct->unk26.x = 0;
+    substruct->unk26.y = 0;
+    substruct->unk26.z = 0;
+    substruct->unk30 = 0.1f;
+    substruct->unk2C = 800;
+    this->actor.flags |= 9;
+    this->actionFunc = func_80B4A5E4;
+}
+#else
 #pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Invadepoh_0x80B439B0/func_80B4A570.asm")
-
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Invadepoh_0x80B439B0/func_80B4A5E4.asm")
-
+#endif
+
+void func_80B4A5E4(EnInvadepoh* this, GlobalContext* globalCtx) {
+    if (this->animPlayFlag != 0) {
+        func_80B4A2C0(this);
+    }
+}
+
+// ISMATCHING: Move rodata once all funcs match
+#ifdef NON_MATCHING
+void func_80B4A614(EnInvadepoh* this) {
+    AlienBehaviorInfo* substruct = &this->behaviorInfo;
+
+    SkelAnime_ChangeAnimTransitionRepeat(&this->skelAnime, &D_06014088, 0.0f);
+    substruct->unk26.x = 0;
+    substruct->unk26.y = 0;
+    substruct->unk26.z = 0;
+    substruct->unk30 = 0.1f;
+    substruct->unk2C = 800;
+    this->actionFunc = func_80B4A67C;
+}
+#else
 #pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Invadepoh_0x80B439B0/func_80B4A614.asm")
-
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Invadepoh_0x80B439B0/func_80B4A67C.asm")
-
+#endif
+
+void func_80B4A67C(EnInvadepoh* this, GlobalContext* globalCtx) {
+    s32 pad;
+
+    Math_StepToF(&this->actor.speedXZ, 5.0f, 1.0f);
+    if (func_80B44B84(this, globalCtx, this->actor.speedXZ, 50.0f)) {
+        func_80B44640(this);
+        this->behaviorInfo.unk4C = 0x5DC;
+        this->actor.speedXZ *= 0.5f;
+    } else {
+        Math_StepToS(&this->behaviorInfo.unk4C, 0x190, 0x32);
+    }
+    func_80B43E6C(this, 6, this->behaviorInfo.unk4C, 0x32);
+    if (this->unk309 == 0) {
+        if (this->unk378 == 0) {
+            this->unk378 = func_80B45550(this, globalCtx, 6400.0f, -0xF);
+        }
+        this->actor.flags &= -0xA;
```

Comment:
Ehh, what?
Did you meant this?
```suggestion
        this->actor.flags &= ~0x9;
```